nemo/collections/llm/gpt/model/base.py

```diff
@@ -171,7 +171,8 @@ class GPTConfig(TransformerConfig, io.IOMixin):
     masked_softmax_fusion: bool = True
     cross_entropy_loss_fusion: bool = True
     gradient_accumulation_fusion: bool = _grad_accum_fusion_available
-    deallocate_pipeline_outputs = True
+    deallocate_pipeline_outputs: bool = True
```

Comment:
Just wonder if these default values have any effect on other classes that subclasses GPTModel